crates/bevy_pbr/src/light/ambient_light.rs

```diff
@@ -20,6 +20,10 @@ use super::*;
 pub struct AmbientLight {
     pub color: Color,
     /// A direct scale factor multiplied with `color` before being passed to the shader.
+    ///
+    /// After applying this multiplier, the resulting value should be in units of [cd/m^2].
```

Comment:
Is it that straightforward? Does it depend on the color type? Is alpha ignored? How can I get the total brightness from each channel?

Reply:
I don't know; I was just following the comments in the issue. @fintelia or @superdump, could you comment on this?

Reply:
First the color gets converted to linear sRGB  (if necessary) and scaled by the brightness:
https://github.com/bevyengine/bevy/blob/4da11fda77f58b9ebe126c92747a6666e530d7d5/crates/bevy_pbr/src/render/light.rs#L985-L986

Then the alpha channel gets completely ignored via a swizzle and the RGB channels are used for rendering calculations:
https://github.com/bevyengine/bevy/blob/4da11fda77f58b9ebe126c92747a6666e530d7d5/crates/bevy_pbr/src/render/pbr_ambient.wgsl#L28

Interpreting the meaning of the three values in the RGB channels is a much deeper rabbit hole. My non-expert understanding is that a value like (100.0, 100.0, 100.0) corresponds to a total light intensity of 100 cd/m^2 divided up as 21 cd/m^2 of red light, 72 cd/m^2 of green light, and 7 cd/m^2 of blue light. Where the colors precise meanings of "red light", "green light", and "blue light" are given by the sRGB spec and the relative weights also come from that same spec

Reply:
Based on its usage in [prepare_lights](https://github.com/bevyengine/bevy/blob/4da11fda77f58b9ebe126c92747a6666e530d7d5/crates/bevy_pbr/src/render/light.rs#L985), the ambient light is always converted to `LinearRgba` (so colour space does not matter), and the colour passed to the GPU is this colour scaled by brightness.

So, we could either say `Color` is unitless (a scalar) and `brightness` is in `cd/m^2`, or the reverse. I _think_ we want to say `Color` is unitless because it conceptually makes more sense that `brightness` is the intensity value (candela/square-metre).